### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc2 in position 7841: invalid continuation byte

### ISSUE THREAD

**Issue by Archurcl4w** [s0md3v/uro#40]
I am using "uro" with other tools to filter duplicate urls, jpegs, etc., and so on. However, after feeding a large wordlist to "uro" with a lot of url paths ending with /678XXXXXXXXXXXXXXXXXXXXXXX, it crashed. Later on, with other normal urls, it no longer works as usual. I installed the uro by pipx-
~~~
pipx install uro
~~~
Steps to reproduce:-
~~~
cat link_results.txt | uro | tee -a output_uro.txt
Traceback (most recent call last):
  File "/usr/local/bin/uro", line 33, in <module>
    sys.exit(load_entry_point('uro==0.0.5', 'console_scripts', 'uro')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/uro-0.0.5-py3.11.egg/uro/uro.py", line 123, in main
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc2 in position 7841: invalid continuation byte
~~~~
In the test environment I have -
uro 1.0.0-beta
Python3 Version - 3.11.9
kali 6.8.11-amd64 kernel
pipx version 1.6.0

It might fix if improve or update in the code regarding UnicodeDecode.

**Comment by Archurcl4w** [comment]
> Hi.
> 
> I've recently have this same issue. If you get this kind of problem and you don't want to complicate with encoding detection.
> 
> The solution is to add this try/except in the file:
> 
> * /usr/local/lib/python3.11/dist-packages/uro/uro.py
> 
> ![uroErrorFixed](https://private-user-images.githubusercontent.com/73143623/308227884-49e12fe1-3a42-43cb-8cf0-392cf37eb274.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4NzcxNTEsIm5iZiI6MTcyMzg3Njg1MSwicGF0aCI6Ii83MzE0MzYyMy8zMDgyMjc4ODQtNDllMTJmZTEtM2E0Mi00M2NiLThjZjAtMzkyY2YzN2ViMjc0LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE3VDA2NDA1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhZmY4ODIwNTlhOGExYTc2ZWRmN2RmMmMyZGNjZGM0MWE5MDUwNTFkNDJiOTI5YzZiOWVhNWUyNzQ3NGM2ZGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.OApb7zoUFxA3FIY0d-B0pv8wxfZA319ooFeYJvKIZPo)
> 
> It will skip the links that have weird encoding, but it will merge all the others that are cool.
> 
> For @s0md3v , please don't leave this amazing project :).

It fixed the issue for time being but the code required to update and improved. Thnaks

**Comment by s0md3v** [comment]
This issue has been fixed in latest release (v1.0.1).
